src/meta/src/controller/streaming_job.rs

```diff
@@ -1412,7 +1411,7 @@ impl CatalogController {
         fragments.retain_mut(|(_, fragment_type_mask, stream_node)| {
             let mut found = false;
             if (*fragment_type_mask & PbFragmentTypeFlag::StreamScan as i32 != 0)
-                || (*fragment_type_mask & PbFragmentTypeFlag::Source as i32 != 0)
```

Comment:
The previous code looks wrong: Alter backfill rate limit will affect MV on non-shared source